### ISSUE TITLE
Time Events don't save between reloads

### ISSUE THREAD

**Issue by WireWhiz** [motion-canvas/motion-canvas#1115]
I'm not sure if this is a bug or a missing feature, but I was going through through building up an animation little by little following an imported audio track, and I was finding that every time I made a code change, the auto-refresh that the browser does resets all of my events to 0. I ended up using a different tool to finish that video, but I would like to use Motion Canvas in the future. Is the expected workflow to finish the code for an animation and then set all the event timings in a single pass before exporting?

**To Reproduce**
Create a yeild* waitUntil() or use a useDuration() in an animation. In the editor set those events to a non zero value. Change anything in the code, such as adding a newline and saving. The editor will now show the events as being set to zero.

**Expected behavior**
I would expect events to persist between reloads, so that I can make changes to an animation without loosing potentially a ton of work. Would also be nice if it persisted indefinitely, so if I go back to a project later it would still have the same timing. 

**Package versions:**
- core: 3.17.0
- two: 3.17.1
- ui: 3.17.0
- vitePlugin: 3.17.0

**Comment by aarthificial** [comment]
Yes, time events should be saved automatically in the `.meta` file located next to your scene and persist between reloads. That's how it works for me and most people.

Is there something specific about your setup? A niche browser or IDE? Something about the file structure? Anything that can help us reproduce this problem would be great.

Additionally, are there any errors in the browser console when you open a project or edit the time events? (You can open the console using `CTRL + SHIFT + J` in Chrome on Windows, for example)

Also, can you verify if it happens in a fresh project created with `npm init @motion-canvas@latest`?

**Comment by WireWhiz** [comment]
Can confirm it happens on two different machines in both an old project and fresh projects. I'm running brave as the browser, error persists with shields on and off. Also checked in Edge and I'm getting the same behavior. No console errors in either the in-editor console or browser console. I can change settings in the editor and see those changes reflected in project.meta, however there's never any event related info saved to [scenename].meta. Using node 18.18.0 at the moment if that helps, I realize there's not really any helpful info here or repo steps, I'll continue poking a bit and see if I can find anything

**Comment by WireWhiz** [comment]
THAT WAS IT, adding `?scene` on to the end of the import path fixed it for me. 
![image](https://github.com/user-attachments/assets/95f3d1b2-eae1-4293-9f38-6037f9eebe92)
Is this documented somewhere and I just missed it? Might be a good idea to add this to a troubleshooting section somewhere, I tried forking gh-pages the repo to do a PR myself but the repo looks like the result of a build command, is there the source somewhere I could contribute to?

**Comment by sysraccoon** [comment]
`?scene` usage already [described in documentation](https://motioncanvas.io/docs/quickstart#explanation)

It might be worth adding a warning when trying to add a scene imported without `?scene`

**Comment by sysraccoon** [comment]
But `tsserver` LSP in neovim already show this as error:
![image](https://github.com/user-attachments/assets/5b80d0f0-3f4e-4d6d-aaa4-7ebea77748e3)
